demo/monitoring-all-process-instances/js/index.js

```diff
@@ -2,12 +2,20 @@
 const timeUseCase = new MonitoringUseCase('time', getHardwareRetailerDiagram, new TimeExecutionData());
 const frequencyUseCase = new MonitoringUseCase('frequency', getHardwareRetailerDiagram, new FrequencyExecutionData());
 
-// Initialize state of radio buttons
+/**
+ * @param {string} useCaseType
+ * @returns {MonitoringUseCase}
+ */
+const detectActualUseCase = useCaseType => useCaseType === 'frequency' ? frequencyUseCase : timeUseCase
+
+// Initialize the state and the radio buttons
+const parameters = new URLSearchParams(window.location.search);
+const useCaseType = ['frequency', 'time'].includes(parameters.get('useCase')) ? parameters.get('useCase') : 'time';
```

Comment:
ℹ️ I am going to try to make the the type property public in the use case class.
This would avoid keeping this variable.

Reply:
done in 29410ece